experimental/sql/dialects/rel_impl.py

```diff
@@ -483,38 +483,30 @@ def get(input: Operation, predicates: Region) -> 'Select':
 
 
 @irdl_op_definition
-class NestedLoopJoin(Operator):
+class NestedLoopCartesianProduct(Operator):
```

Comment:
Why not call this simply `CartesianProduct` here as well? There is no other (typical) way to compute the Cartesian product than a nested loop, right? That distinction makes sense for the join because there are many other variants...